src/support/HISTORY.md

```diff
@@ -1,6 +1,10 @@
 Release History
 ===============
 
+2.0.0
+-----
+* New File Attachment feature added to support ticket 
```

Comment:
Its not a new file attachment feature, we are changing everything. Just write something like CLI update to use new Support RP API or something for now